docs/src/man/joins_new.md

```diff
@@ -0,0 +1,879 @@
+# Joining DataFrames
+
+"You may have two different data sets in Julia but you may not be sure how to perform *joins*. What 
+criteria should I consider? What are the different ways I can join these data sets?"
+
+!!! note
+
+    Some of the part has been taken from [Bence Komarniczky](https://towardsdatascience.com/joining-dataframes-in-julia-c435e3da32f3)
+
+Sound familiar? You may have come across this question plenty of times on online discussion
+forums. Working with one data frame is fairly straightforward but things become challenging when
+we have data spread across two or more data frames. This is where the concept of *joins* come in.
+
+*Joins* is a very common and important operation that arises in the world of tabulated data. 
+A join across two data frames is the action of combining the two datasets based on shared column values 
+that exist across them. We call this column (or columns) the *key(s)*. So, each record from the 
```

Comment:
we can keep keys and records terminology, but we do not use it in DataFrames.jl documentation. Where have you taken it from?